plugins/woocommerce-admin/client/products/utils/get-product-stock-status.ts

```diff
@@ -0,0 +1,49 @@
+/**
```

Comment:
Hopefully this util isn't pushing the scope of this PR too far.  Happy to separate it into another PR if it helps.